cowprotocol/cowswap
# Fix circular deps for cosmos

# Summary

Fixes cosmos breaking on dev due to circular deps.

  # To Test

On your local;
1. Run `yarn cosmos`
2. Go to `localhost:5001`
3. You should be able to access cosmos instead of seeing a reference error.

  # Background

https://cowservices.slack.com/archives/C0361CDG8GP/p1679397502020789
